dappnode_package.json

```diff
@@ -25,7 +25,7 @@
   "requirements": {
     "minimumDappnodeVersion": "0.2.60"
   },
-  "categories": ["Blockchain", "ETH2.0", "Gnosis"],
+  "categories": ["Blockchain"],
```

Comment:
Is there a specific reason for this change?

Reply:
I was unable to build a package (most likely `"Gnosis"` category is not supported), and I just copied categories list from other dappnode packages for gnosis, e.g.:
https://github.com/dappnode/DAppNodePackage-Lodestar-Gnosis/blob/main/dappnode_package.json#L16
https://github.com/dappnode/DAppNodePackage-teku-gnosis/blob/master/dappnode_package.json#L15

Should be relatively easy to add `Gnosis` to categories enum btw